app/main.py

```diff
@@ -1,8 +1,48 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        total_wash_price = 0
+        for car in cars:
+            if self.wash_single_car(car):
+                total_wash_price += self.calculate_washing_price(car)
+                car.clean_mark = self.clean_power
```

Comment:
Clean Code issue: According to the checklist, you can avoid else when you have a return statement. In your 'serve_cars' method, you have an if statement that only executes code when the condition is true. To make your code cleaner, consider removing the if statement and returning early if the condition is not met. Here's how you might do it:

python
for car in cars:
    if not self.wash_single_car(car):
        continue
    total_wash_price += self.calculate_washing_price(car)
    car.clean_mark = self.clean_power
```